cli/cmd/org/delete.go

```diff
@@ -11,27 +12,24 @@ import (
 
 func DeleteCmd(ch *cmdutil.Helper) *cobra.Command {
 	var force bool
-	var name string
 
 	deleteCmd := &cobra.Command{
 		Use:   "delete [<org-name>]",
```

Comment:
Since you removed the flag, the positional arg is no longer optional:

```suggestion
		Use:   "delete <org-name>",
```

Reply:
Is this preferred across all arguments?